scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2696,69 +2696,69 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
 
   def entrypointModifier(): Mod = {
-    val mod = autoPos {
-      def annot() = annots(skipNewLines = false) match {
-        case annot :: Nil => annot
-        case _ :: other :: _ =>
-          syntaxError(s"end of file expected but ${token.name} found", at = other)
-        case Nil => unreachable
-      }
-      def fail() =
-        syntaxError(s"modifier expected but ${parser.token.name} found", at = parser.token)
-      token match {
-        case t: Unquote => unquote[Mod](t)
-        case At() => annot()
-        case KwPrivate() => privateModifier()
-        case KwProtected() => protectedModifier()
-        case KwImplicit() => next(); Mod.Implicit()
-        case KwFinal() => next(); Mod.Final()
-        case KwSealed() => next(); Mod.Sealed()
-        case KwOverride() => next(); Mod.Override()
-        case KwCase() => next(); Mod.Case()
-        case KwAbstract() => next(); Mod.Abstract()
-        case Ident("+") => next(); Mod.Covariant()
-        case Ident("-") => next(); Mod.Contravariant()
-        case KwLazy() => next(); Mod.Lazy()
-        case KwVal() if !dialect.allowUnquotes => next(); Mod.ValParam()
-        case KwVar() if !dialect.allowUnquotes => next(); Mod.VarParam()
-        case soft.KwOpen() => next(); Mod.Open();
-        case soft.KwTransparent() => next(); Mod.Transparent()
-        case soft.KwInline() => next(); Mod.Inline()
-        case soft.KwInfix() => next(); Mod.Infix()
-        case Ident("valparam") if dialect.allowUnquotes => next(); Mod.ValParam()
-        case Ident("varparam") if dialect.allowUnquotes => next(); Mod.VarParam()
-        case _ => fail()
-      }
+    val mod = token match {
+      case t: Unquote => unquote[Mod](t)
+      case At() =>
+        annots(skipNewLines = true) match {
+          case annot :: Nil => annot
+          case _ :: other :: _ =>
+            syntaxError(s"end of file expected but ${token.name} found", at = other)
+          case Nil => unreachable
+        }
+      case KwPrivate() => privateModifier()
+      case KwProtected() => protectedModifier()
+      case KwImplicit() => atCurPosNext(Mod.Implicit())
+      case KwFinal() => atCurPosNext(Mod.Final())
+      case KwSealed() => atCurPosNext(Mod.Sealed())
+      case KwOverride() => atCurPosNext(Mod.Override())
+      case KwCase() => atCurPosNext(Mod.Case())
+      case KwAbstract() => atCurPosNext(Mod.Abstract())
+      case Ident("+") => atCurPosNext(Mod.Covariant())
+      case Ident("-") => atCurPosNext(Mod.Contravariant())
+      case KwLazy() => atCurPosNext(Mod.Lazy())
+      case KwVal() if !dialect.allowUnquotes => atCurPosNext(Mod.ValParam())
+      case KwVar() if !dialect.allowUnquotes => atCurPosNext(Mod.VarParam())
+      case Ident("valparam") if dialect.allowUnquotes => atCurPosNext(Mod.ValParam())
+      case Ident("varparam") if dialect.allowUnquotes => atCurPosNext(Mod.VarParam())
+      case _ =>
+        token.toString match {
```

Comment:
compared to `modifier()` (see line 2715), this `entrypointModifier` doesn't have `opaque`; is that correct?

Reply:
This might be wrong, thought this was never tested, which might explain why this never popped up.